backend/backend/backend-src/lib/events.ts

```diff
@@ -2,10 +2,45 @@
  * @file Handlers related to events and balances.
  */
 
-import {Request, Response} from 'express'
+import {Request, Response, Router} from 'express'
 import * as Checks from './checks'
 import * as dbEvents from './db/events'
 
+export function createEventsRouter(): Router {
+  const router = Router()
+  router.get('/:address/:ticker', async (req, res, next) => {
```

Comment:
Is there a way to avoid writing try/catch everywhere?  Maybe with some middleware on router?

Reply:
Yes, I actually did it this way before 
` const catchError = (handler: RequestHandler): RequestHandler => async (req, res, next) => {
    try {
      await handler(req, res, next)
    } catch (error) {
      next(error)
    }
  }

  router.use((req: Request, res, next) => {
    req.tokenPath = TOKEN_PATH
    req.workDir = WORK_DIR
    next()
  })`
  
  
  but the current approach gives more control if we want to customize errors for each API in the future.

Reply:
Can you use the implementation in #91?